production/scrapers/lasd.R

```diff
@@ -46,154 +46,185 @@ lasd_pull <- function(x, wait = 5){
 }
 
 lasd_restruct <- function(x){
+    
+    # Trim whitespace and brighten image 
+    x <- magick::image_trim(x) %>% 
+        magick::image_modulate(brightness = 120)
```

Comment:
Relevant changes here to (1) trim whitespace off the image, and (2) brighten it up so the black text on purple reads a little better.